core/templates/dev/head/services/contextual/UrlService.js

```diff
@@ -34,10 +38,28 @@ oppia.factory('UrlService', ['$window', function($window) {
       return urlParts[1] === 'embed';
     },
     getPathname: function() {
-      return $window.location.pathname;
+      return this.getCurrentUrl().pathname;
+    },
+    getParamValuesAsList: function(queryTag) {
```

Comment:
Maybe call this getQueryFieldValuesAsList and call the arg fieldName -- a "tag" kind of means a different thing in HTML-land. (I thought it was param before, sorry, but then I looked it up and "field name", "field value" seems to be the terminology -- https://en.wikipedia.org/wiki/Query_string.)

Reply:
Done